players/Randomplayer.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+
+from .Player import Player
+import random
+
+
+class Randomplayer(Player):
+
+    """A class representing a random player.
+        A random player who always guess randomly with the flipped coin game. 
```

Comment:
* Should work for any game, not just CoinFlip
* Let's keep this super tidy. Please add a line break to it's <80 characters wide
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
Do you use any particular code formatter to do this or you just manually format it to make sure it doesn't go over the minimum line length?

Reply:
I use pyflakes embedded into my text editor. Maybe I also use flake8. It checks the python you write conforms to pep8 standard